test/e2e-migrated/logs/fluentbit/modify_timestamp_date_format_test.go

```diff
@@ -54,7 +55,7 @@ func TestModifyTimestampDateFormat(t *testing.T) {
 	assert.FluentBitLogPipelineHealthy(t.Context(), suite.K8sClient, pipelineName)
 	assert.DaemonSetReady(t.Context(), suite.K8sClient, kitkyma.FluentBitDaemonSetName)
 	assert.DeploymentReady(t.Context(), suite.K8sClient, backend.NamespacedName())
-	assert.DeploymentReady(t.Context(), suite.K8sClient, logProducer.NamespacedName())
+	assert.DeploymentReady(t.Context(), suite.K8sClient, types.NamespacedName{Name: stdloggen.DefaultName, Namespace: genNs})
```

Comment:
Is this change needed?

Reply:
As the producer has no state anymore, I cannot easily provide this function. Any better idea?